Hypeclass/API/SearchManager.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SearchManager.swift
+//  Hypeclass
+//
+//  Created by Hong jeongmin on 2022/07/26.
+//
+
+import Foundation
+import FirebaseFirestore
+import FirebaseFirestoreSwift
+
+class SearchManager {
+    static let shared = SearchManager()
+    
+    func requestSearchDancer(queryString query: String) async throws -> [Dancer]? {
+        let snapshot = try await Constant.dancerRef.whereField("name", isGreaterThanOrEqualTo: query).getDocuments()
+        
+        let searchDancer = snapshot.documents.compactMap { document in
+            try? document.data(as: Dancer.self)
+        }
+        
+        return searchDancer
+    }
+    
+    func requestSearchStudio(queryString query: String) async throws -> [Studio]? {
```

Comment:
SearchManager 안에있는 메서드라서 Search 는 빼도 유추가능할 것 같습니다 ㅎ
```suggestion
    func requestStudio(queryString query: String) async throws -> [Studio]? {
```